app/services/proofing/mock/ddp_mock_client.rb

```diff
@@ -32,6 +32,11 @@ class DdpMockClient < Proofing::Base
 
       proof do |applicant, result|
         result.transaction_id = TRANSACTION_ID
+        result.response_body = File.read(
+          Rails.root.join(
+            'spec', 'fixtures', 'proofing', 'lexis_nexis', 'ddp', 'successful_response.json'
+          ),
+        )
```

Comment:
should we only do this in the `pass` case? failures should get different responses IMO

Reply:
⛔ 💡 this needs to have a `JSON.parse` ... the real and mock proofers are setting different values (see https://github.com/18F/identity-idp/pull/6857/files#r957501496)

(Also we can use `Pathname#read` here)
(but I still think this should only be in the success/pass case)

```suggestion
        result.response_body = JSON.parse(
          Rails.root.join(
            'spec', 'fixtures', 'proofing', 'lexis_nexis', 'ddp', 'successful_response.json'
          ).read,
        )
```

Reply:
pushed a generic review_status fail state for the forced error states in the mock...

Reply:
yes great catch!  i was going to run the live calls on my local to validate everything was working like the mock but didn't get there yet.

Reply:
i still plan on doing that but again great catch.